admin_guide/scheduling/scheduler.adoc

```diff
@@ -62,33 +269,122 @@ These predicates do not take any configuration parameters or inputs from the
+----
+{"name" : "MatchInterPodAffinity"}
+----
+
+**_NoDiskConflict_** checks if the volume requested by a pod is available.
+_The default scheduler policy includes this predicate._
+----
+{"name" : "NoDiskConflict"}
+----
+
+**_PodToleratesNodeTaints_** checks if a pod can tolerate the node taints.
+_The default scheduler policy includes this predicate._
+----
+{"name" : "PodToleratesNodeTaints"}
+----
+
+**_CheckNodeMemoryPressure_** checks if a pod can be scheduled on a node with a memory pressure condition.
+_The default scheduler policy includes this predicate._
+----
+{"name" : "CheckNodeMemoryPressure"}
+----
+
+**_CheckNodeDiskPressure_** checks if a pod can be scheduled on a node with a disk pressure condition.
+_The default scheduler policy includes this predicate._
+----
+{"name" : "CheckNodeDiskPressure"}
+----
+
+**_NoVolumeNodeConflict_**
+----
+{"name" : "NoVolumeNodeConflict"}
+----
+
+**_CheckVolumeBinding_** Evaluates if a pod can fit based on the volumes, it requests, for both bound and unbound PVCs.
+* For PVCs that are bound, the predicate checks that the corresponding PV's node affinity is satisfied by the given node.
+* For PVCs that are unbound, the predicate searched for available PVs that can satisfy the PVC requirements and that
+the PV node affinity is satisfied by the given node.
+
+The predicate returns true if all bound PVCs have compatible PVs with the node, and if all unbound PVCs can be matched with an available and node-compatible PV.
+----
+{"name" : "CheckVolumeBinding"}
+----
+
+The `CheckVolumeBinding` predicate must be enabled in non-default schedulers.
+
+**_CheckNodeCondition_** checks if a pod can be scheduled on a node reporting *out of disk*, *network unavailable*, or *not ready* conditions.
+----
+{"name" : "CheckNodeCondition"}
+----
+
+**_PodToleratesNodeNoExecuteTaints_** checks if a pod tolerations can tolerate a node *NoExecute* taints.
+----
+{"name" : "PodToleratesNodeNoExecuteTaints"}
+----
+
+**_CheckNodeLabelPresence_** checks if all of the specified labels exist on a node, regardless of their value.
+----
+{"name" : "CheckNodeLabelPresence"}
+----
+
+**_checkServiceAffinity_** checks that ServiceAffinity labels are homogeneous for pods that are scheduled on a node.
 ----
-{"name" : "PodFitsPorts"}
+{"name" : "checkServiceAffinity"}
 ----
 
-**_PodFitsResources_** determines a fit based on resource availability. The
+**_MaxAzureDiskVolumeCount_**  checks the maximum number of Azure Disk Volumes.
+----
+{"name" : "MaxAzureDiskVolumeCount"}
+----
+
+[[admin-guide-scheduler-general-predicates]]
+=== General Predicates
+
+The following general predicates check whether noncritical predicates and essential predicates pass. Noncritical predicates are the predicates
+that only non-critical pods need to pass and essential predicates are the predicates that all pods need to pass.
+
+_The default scheduler policy includes the general predicates._
+
+
+**Non-critical general predicates**
```

Comment:
Consider using `====` or `====` for this and the "Essential general predicates", with or without a leading `discrete`, instead of bold markup. I.e.:

```
[discrete]
==== Non-critical general predicates
```